v2/internal/runner/runner.go

```diff
@@ -405,6 +405,15 @@ func (r *Runner) RunEnumeration() error {
 		}
 		return nil // exit
 	}
+
+	// parse the loaded templates if only uncover flag is passed
+	// add the hosts from the metadata queries into input provider
+	if r.options.Uncover && len(r.options.UncoverQuery) == 0 {
```

Comment:
Instead of doing this here, cant use use load in the below line to load the templates and then process them for metadata?

Reply:
removed the template parsing logic from the `GetUncoverTargetsFromMetadata`. 
Now it accepts the already parsed templates by `store.load()` method.